.gitignore

```diff
@@ -1,3 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```

Comment:
We need to keep this the rest are repeats I think

Reply:
I left the items from original .gitignore for a reason - hence the commented line for pre-2023. Those were things from the prior project that we may re-use. There is no issue with leaving them as is. Feel free to inquire about this in Discord for discussion with the rest of the Core Team Admins and other volunteers.

I did miss a few of the duplicates, so thank you for that. Resolving them directly in the PR.

Reply:
You want to remove them because it might cause issues like we did today. I feel like if you need them later you can just add them then?